components/User/SessionBox.tsx

```diff
@@ -0,0 +1,74 @@
+import { Drawer, List, ListItem, ListItemButton, ListItemText } from '@mui/material';
+import { observable } from 'mobx';
+import { observer } from 'mobx-react';
+import Link from 'next/link';
+import { Component, HTMLAttributes, JSX } from 'react';
+
+import { PageHead } from '../PageHead';
+import { SessionForm } from './SessionForm';
+
+export type MenuItem = Pick<JSX.IntrinsicElements['a'], 'href' | 'title'>;
+
+export interface SessionBoxProps extends HTMLAttributes<HTMLDivElement> {
+  path?: string;
+  menu?: MenuItem[];
+  jwtPayload?: any; // TODO: Define proper JWT payload type
+}
```

Comment:
```suggestion
export interface SessionBoxProps extends HTMLAttributes<HTMLDivElement>, JWTProps<User> {
  path?: string;
  menu?: MenuItem[];
}
```
使用 `@idea2app/data-server` GitHub NPM 包中的类型